test/built-ins/Promise/any/iter-arg-is-false-reject.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2019 Sergey Rubanov. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-promise.any
+description: >
+  Promise.any(false) rejects with TypeError.
+info: |
+  Promise.any ( iterable )
+
+  ...
+  3. Let iteratorRecord be GetIterator(iterable).
+  4. IfAbruptRejectPromise(iteratorRecord, promiseCapability).
+  ...
+
+  #sec-getiterator
+  GetIterator ( obj [ , hint [ , method ] ] )
+
+  ...
+  Let iterator be ? Call(method, obj).
+  If Type(iterator) is not Object, throw a TypeError exception.
+  ...
+features: [Promise.any, Symbol.iterator]
```

Comment:
same for the usage of Symbol.iterator, originals from https://github.com/tc39/test262/pull/1496, I believe we don't need to "require" this feature as we expect the results the same way.